docs/pages/pmd/userdocs/help.md

```diff
@@ -1,8 +1,26 @@
 ---
 title: Getting Help
-sidebar: pmd_sidebar
 permalink: pmd_userdocs_help.html
-folder: pmd/userdocs
+author: Andreas Dangel <andreas.dangel@adangel.org>
+last_updated: September 2017
 ---
 
-Asking questions, reporting bugs, ...
+There are numerous ways of getting help:
+
+*   Browse already existing questions on [StackOverflow](https://stackoverflow.com/questions/tagged/pmd),
+    that are [tagged with `pmd`](https://stackoverflow.com/questions/tagged/pmd).
+
+*   You can search on [StackOverflow](https://stackoverflow.com/) for a similar problem or ask a new question there.
```

Comment:
asking to search after browsing seems a little redundant... isn't it? We can probably just ask the user to search for questions on stackoverflow, and if posting a new one, tag it properly

Reply:
changed